api/rest/restapi.go

```diff
@@ -585,6 +585,62 @@ func (api *API) allocationHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// defined groups / aliases for filtering
+var filterGroups = map[string][]string{
+	"error":  {"error", "pin_error", "unpin_error", "cluster_error"},
+	"queued": {"queued", "pin_queued", "unpin_queued"},
+}
+
+func parseAllFilters(rawFilters string) map[string]bool {
+	// given the initial string of filters, return a slice of all individual strings
+	// to match on. This includes replacing alises
+	rawFilterList := strings.Split(strings.ToLower(rawFilters), ",")
+	var filterList = map[string]bool{}
+
+	for _, rawFilter := range rawFilterList {
+		if aliasSlice, exists := filterGroups[rawFilter]; exists {
+			for _, alias := range aliasSlice {
+				filterList[alias] = true
+			}
+		} else {
+			filterList[rawFilter] = true
+		}
+	}
+	return filterList
+
+}
+
+func filterStatusAll(filter string, pinInfos []types.GlobalPinInfoSerial) []types.GlobalPinInfoSerial {
+	if filter == "" {
+		return pinInfos
+	}
+	filterList := parseAllFilters(filter)
+	var filtered []types.GlobalPinInfoSerial
+NEXTPI:
+	for _, pi := range pinInfos {
+		for _, entry := range pi.PeerMap {
+			if _, exists := filterList[entry.Status]; exists {
```

Comment:
`filterList` is a `map[string]bool` where the `bool` value is only true if it does exist, so this if statement can be shortened to the following:
```go
if filterList[entry.Status] {
```

Reply:
This is the same in `filterStatus`.